### ISSUE TITLE
When enabled JAVADOC_BANNER=YES, the reference line#s for functions are incorrect (shifted some lines) 

### ISSUE THREAD

**Issue by ian2009** [doxygen/doxygen#7441]
[doxygen_javadoc_issue.zip](https://github.com/doxygen/doxygen/files/3956677/doxygen_javadoc_issue.zip)

Version: 1.8.16.

I have an example C++ cpp file ("javadoc-banner.cpp" in the attached) of JAVADOC BANNER styled comments like:
```
/***********************************
* comments
************************************/
```
In the Doxygen file, set JAVADOC_BANNER=YES, the parsing is fine, but the reference line#s for functions are incorrect (refer to the "Capture.png" in the attached); if I changed the comment styles to:
 ```
/**
* comments
********************************/
```
The reference line#s will be correct.

**Comment by albert-github** [comment]
With version 1.8.16 I do see the problem with the current master version it is gone and I get:

![Line_no_master](https://user-images.githubusercontent.com/5223533/70733872-74399d00-1d0b-11ea-9a86-611de6ef7e68.jpg)

I've seen some corrections regarding line counting (part of other issue as well), but with a quick search  I cannot pinpoint to the relevant fix / issue.

But I think the  the problem is Fixed but not released.

**Comment by ian2009** [comment]
Great! Thanks a lot for the quick reply!

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.17.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
